pages/linux/wdctl.md

```diff
@@ -0,0 +1,16 @@
+# wdctl
+
+> Shows the hardware watchdog status.
+> More information: <https://manned.org/wdctl>.
+
+- Display the watchdog status:
+
+`wdctl`
+
+- Display the watchdog status in a single line in `key=value` pairs:
```

Comment:
I not sure if specific the format in the example description, maybe this is better
```suggestion
- Display the watchdog status in a single line in key-value pairs:
```